src/ticket/controllers/ticket.controller.ts

```
@@ -1,25 +1,95 @@
+  @ApiBearerAuth('admin')
+  @ApiBearerAuth('user')
+  @ApiOperation({
+    summary: 'Create ticket',
+    description: 'Create ticket of source in user account',
+  })
+  @ApiForbiddenResponse()
+  @ApiOkResponse()
+  @Post()
+  create(
+    @Req() req: Request & { user: UniversalJwtRequest },
+    @Body() createTicketDto: CreateTicketDto,
```

ESLint

[error] 'createTicketDto' is defined but never used.

(@typescript-eslint/no-unused-vars)